test/e2e/systemd_test.go

```diff
@@ -77,4 +79,39 @@ WantedBy=multi-user.target
 		status := SystemExec("bash", []string{"-c", "systemctl status redis"})
 		Expect(status.OutputToString()).To(ContainSubstring("active (running)"))
 	})
+
+	It("podman run container with systemd PID1", func() {
+		systemdImage := "fedora"
+		pull := podmanTest.Podman([]string{"pull", systemdImage})
+		pull.WaitWithDefaultTimeout()
+		Expect(pull.ExitCode()).To(Equal(0))
+
+		ctrName := "testSystemd"
+		run := podmanTest.Podman([]string{"run", "--name", ctrName, "-d", systemdImage, "init", "--log-level", "debug"})
```

Comment:
IIUC, this line requires `oci-systemd-hook` to be instalelld on system.

What do you think about another test-case with without that hook? (or with `env oci-systemd-hook=disabled`)

Reply:
podman should not require oci-systemd-hook at all.

Reply:
ok; tahnk you for explanation.